jetbrains-core/resources/telemetryOverride.json

```diff
@@ -1,5 +1,11 @@
 {
-    "types": [],
+    "types": [
+        {
```

Comment:
probably fix and move into common

Reply:
https://github.com/aws/aws-toolkit-common/pull/198

Reply:
done